pyspecProcScripts/third_level/process_enhancement.py

```diff
@@ -27,13 +27,14 @@ def select_pathway(s,pathway):
     retval = s
     for k,v in pathway.items():
         retval = retval[k,v]
-    return retval    
+    return retval
 # slice out the FID from the echoes,
 # also frequency filtering, in order to generate the
 # list of integrals for ODNP
 # to use: as a rule of thumb, make the white boxes
 # about 2x as far as it looks like they should be
 # leave this as a loop, so you can load multiple files
+# why is this part of this pull request??
```

Comment:
note this

Reply:
I'm skipping most things with this file b/c it shouldn't be part of this pull request almost definitely, and I also think will be superseded by the table of integrals.

Reply:
these are part of the pull request because they use correlation alignment and so they need to be edited since we are only using one cohesive alignment script instead of two now.

Reply:
The purpose of this PR is to combine the two correlation alignment scripts into one. And so along with that I need to modify the scripts that call correlation alignment to edit accordingly

Reply:
see slack